pkg/capture/capturetypes/status.go

```diff
@@ -33,6 +33,7 @@ func (cs State) String() string {
 // PacketStats stores the packet statistics of the capture
 type PacketStats struct {
 	*CaptureStats
+	PacketsCapturedOverall int
```

Comment:
I found this to be very helpful (and required for E2E tests), maybe there's better ways than this, but we need to know how many packets have been processed since the beginning of time (or at least creation of this capture instance), otherwise there's no way to know when to stop the E2E test (and fetching that data reliably via the rotations is impossible at the moment). Probably something to improve for the refactor.

Reply:
I'm totally on board with this. 

Another application for this: in the current `gpctl status` target, it's really confusing that the packets aren't monotonically increasing and only the difference is shown.